cicd/forgeops-tests/forgeops-tests.py

```diff
@@ -16,7 +21,42 @@
 # Insert config folder (rank 1) to python path
 sys.path.insert(0, os.path.join(root_dir, 'config'))
 
-import pytest
+
+def set_allure_environment_props(filename):
+    # Get os environment properties as dictionary
+    environment_properities = dict(os.environ)
+
+    # Read in the properties currently specified in environment.properties
+    current_properties = {}
+    if os.path.exists(filename):
+        with open(filename, 'r') as file:
+            for line in file:
+                line = line.rstrip()
+                if "=" not in line: continue
+                if line.startswith("#"): continue
+                key, value = line.split("=", 1)
+                current_properties[key] = value
+
+    # Remove properties contained previously in current properties that are not specified in environment properties
```

Comment:
Not sure to understand in which case you need to remove values from the current property file.

Reply:
The case I was thinking of was environment.properties contains something like TESTS_EXPO=blah because it was one of the environment properties set in the previous test.  But, in subsequent test TESTS_EXPO isn't set as an environment property.  So, I don't want to include it for this test report.